Lib/test/test___all__.py

```diff
@@ -104,7 +104,7 @@ def test_all(self):
         # In case _socket fails to build, make this test fail more gracefully
         # than an AttributeError somewhere deep in concurrent.futures, email
         # or unittest.
-        import _socket
+        import _socket  # noqa: F401
```

Comment:
Would not using `import_module()` be better?

Reply:
The intent is to get an error if the module is missing, not to skip the test.